### PR TITLE
Miscellaneous improvements

### PR DESCRIPTION
**Pull request by tfussell**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,6 +5,13 @@ WORKDIR /go/gsctl
 RUN git checkout add-json-output
 RUN CGO_ENABLED=0 go build
 
+FROM quay.io/giantswarm/alpine:3.12 AS kubectl
```
Comment (tfussell) [inline Dockerfile:8]: It turns out `gsctl create kubeconfig` requires `kubectl`.

File: cmd/create/runner.go [hunk 2]
```diff
@@ -162,7 +164,7 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 	var release v1alpha1.Release
 	var provider string
 	var releaseVersion string
-	r.logger.LogCtx(context.Background(), "message", "determining release to test")
```
Comment (tfussell) [inline cmd/create/runner.go:165]: Took the opportunity to use the actual context instead of `context.Background()`.

File: cmd/create/runner.go [hunk 3]
```diff
@@ -272,15 +275,81 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
...
 
-	// Write to path from flag
-	err = ioutil.WriteFile(r.flag.OutputPath, []byte(kubeconfig), 0644)
+	r.logger.LogCtx(ctx, "message", fmt.Sprintf("writing cluster ID to path %s", r.flag.OutputClusterID))
+	err = ioutil.WriteFile(r.flag.OutputClusterID, []byte(clusterID), 0644)
 	if err != nil {
 		return microerror.Mask(err)
 	}
 
+	if r.flag.Wait {
```
Comment (tfussell) [inline cmd/create/runner.go:292]: Rudimentary readiness check. We can use a more robust function, library (`tenantcluster`?), or binary (`awscnfm`?) later if desired.

**Review by yulianedyalkova — APPROVED** [review]

**Review by MarcelMue — APPROVED** [review]
Let's merge this and iterate I would say.

**Pull request merged**